packages/livekit-rtc/src/audio_source.ts

```diff
@@ -47,7 +63,46 @@ export class AudioSource {
     this.ffiHandle = new FfiHandle(res.source.handle.id);
   }
 
+  get queuedDuration(): number {
+    return Math.max(this.currentQueueSize - Date.now() + this.lastCapture, 0);
```

Comment:
Date.now isn't monotonic, can you use https://nodejs.org/api/process.html#process_process_hrtime_bigint